keyboards/telophase/readme.md

```diff
@@ -0,0 +1,24 @@
+Mitosis
+=======
+
+A wireless split compact keyboard.
+
+Keyboard Maintainer: [@reversebias](https://github.com/reversebias]
```

Comment:
Make sure that you have a double space on each of these lines, or they end up as one, long line.